### PR TITLE
As-card4-song-list

### PR DESCRIPTION
**Pull request by ASlayton**
TO test: Pull and look at main.js to see object

### CODE REVIEW

File: javascript/main.js [hunk 1]
```diff
@@ -1,7 +1,154 @@
...
+    audioURL: "audioclips/audio_hero_MonsterEat_DIGIJ03-57-2.mp3"
+  },
+  {
+    songId: 20,
+    album: "Single",
+    name: "Playtime",
+    duration: 105,
+    audioURL: "audioclips/Alash_Ensemble_-_01_-_Syyn_Miygak.mp3"
+  }
+]
```
Comment (amillion3) [inline javascript/main.js]: missing semicolon

File: javascript/main.js [hunk 2]
```diff
@@ -1,7 +1,154 @@
+
+//SONG LIST OBJECT ARRAY
+var Songs = [
```
Comment (amillion3) [inline javascript/main.js]: change to lowercase 's'

**Comment by ASlayton** [comment]
Changes have been made

**Comment by aprilrochelle** [comment]
👍 Looks good to me

**Comment by amillion3** [comment]
👍 Lookin good!

**Pull request merged**